format/format.go

```diff
@@ -81,13 +81,21 @@ var companyNameEndings = [...]string{
 	"UNLTD",
```

Comment:
Company Name Endings would be better placed in [api-enumerations](https://github.com/companieshouse/api-enumerations)

Reply:
Completely agree. Owing to time constraints, it's not going to be possible right now. @Mathsh, can we raise this as an issue against the repo?

Reply:
Yes, Brad is correct. This is an internal tool for the support team to use to load data from mongo to ES7. It will be improved on as we go.

Reply:
Happy for this not to be done at this time, as you have not introduced the change. But it is important that it does not get forgotten.